src/strategies/staked-keep/index.ts

```diff
@@ -0,0 +1,45 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const UNISWAP_SUBGRAPH_URL = {
```

Comment:
Maybe rename to `KEEP_SUBGRAPH_URL` :)

Reply:
Will do, that's an artifact of the code I cloned